test/http/nominatim.yml

```diff
@@ -1,12 +1,12 @@
-/search?accept-language=&format=xml&q=Hoddesdon&viewbox=-0.559%2C51.766%2C0.836%2C51.217:
+/search?accept-language=&extratags=1&format=xml&q=Hoddesdon&viewbox=-0.559,51.766,0.836,51.217:
```

Comment:
So was this actually failing and hitting the live nominatim before if the stubbed URL was no longer correct?